### PR TITLE
Add Cat generator

### PR DESCRIPTION
**Pull request by 51d4**
Hi @vadymmarkov ,

I added my favourite generator class - Cat

Please take a look :)

### CODE REVIEW

**Comment by dunyakirkali** [comment]
@jeroendevrind Could you also update the `README.md` with some usage instructions?

File: Sources/Fakery/Generators/Cat.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//
```
Comment (dunyakirkali) [inline Sources/Fakery/Generators/Cat.swift]: @51d4 Could u please remove the copyright?

Comment (51d4) [reply Sources/Fakery/Generators/Cat.swift]: Good catch! Thanks :)

**Comment by vadymmarkov** [comment]
Nice one @51d4! Could you also add this generator to https://github.com/vadymmarkov/Fakery/blob/master/Sources/Fakery/Faker.swift to make it accessible.

**Pull request merged**